.stylelintrc

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
So here is just basic setup, there are several plugins that we can consider, but I'd rather consider and integrate them in a separate PR (bem-naming and combinator-selectors rules to name a few).